src/main/java/ua/danit/final_project/services/WorkCommentServiceImpl.java

```diff
@@ -0,0 +1,80 @@
+package ua.danit.final_project.services;
+
+import org.joda.time.DateTime;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import ua.danit.final_project.entities.ShiftComment;
+import ua.danit.final_project.entities.User;
+import ua.danit.final_project.entities.WorkShift;
+import ua.danit.final_project.repositories.ShiftCommentRepository;
+import ua.danit.final_project.repositories.WorkShiftRepository;
+
+import javax.persistence.EntityNotFoundException;
+import java.sql.Timestamp;
+import java.util.Date;
+import java.util.List;
+
+@Service
+public class WorkCommentServiceImpl implements WorkCommentService {
+
+  private final WorkShiftRepository workShiftRepository;
+  private final ShiftCommentRepository shiftCommentRepository;
+
+  @Autowired
+  public WorkCommentServiceImpl(WorkShiftRepository workShiftRepository,
+                                ShiftCommentRepository shiftCommentRepository) {
+    this.workShiftRepository = workShiftRepository;
+    this.shiftCommentRepository = shiftCommentRepository;
+  }
+
+  @Override
+  public List<ShiftComment> getShiftCommentsByDate(Long miliseconds) {
+    Timestamp date = new Timestamp(miliseconds);
+    DateTime searchDate = new DateTime(date).withTimeAtStartOfDay();
+    Date from = searchDate.toDate();
+    Date to = searchDate.plusHours(24).toDate();
+    return shiftCommentRepository.findAllByDateBetween(from, to);
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Replace this use of System.out or System.err by a logger. ([squid:S106](https://sonarcloud.io/organizations/final-project/rules#rule_key=squid%3AS106))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=final-project&pullRequest=85&issues=AWWG_p8PX2sY87p04Y8D&open=AWWG_p8PX2sY87p04Y8D)

[ISSUE_KEY]: AWWG_p8PX2sY87p04Y8D